docs/admin/src/widgets/StringTrimmed.js

```diff
@@ -1,39 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default class StringTrimmedControl extends React.Component {
-  static propTypes = {
-    onChange: PropTypes.func.isRequired,
-    forID: PropTypes.string,
-    value: PropTypes.node,
-    classNameWrapper: PropTypes.string.isRequired,
-    setActiveStyle: PropTypes.func.isRequired,
-    setInactiveStyle: PropTypes.func.isRequired,
-  };
+export default function StringTrimmedControl({
+  onChange,
+  forID,
+  value,
+  classNameWrapper,
+  setActiveStyle,
+  setInactiveStyle,
+}) {
+  const [state, setState] = React.useState({ value: value || '' });
```

Comment:
If you're writing a lot of code that will use any of these react /redux hooks, you can also add this to the imports at the top

import React, { useState } from 'react';

so you can just do

const [foo, setFoo] = useState('');